src/inflight_activation_store.rs

```diff
@@ -203,6 +209,7 @@ impl InflightActivationStore {
                 b.push_bind(row.offset);
                 b.push_bind(row.added_at);
                 b.push_bind(row.remove_at);
+                b.push_bind(row.expires_at);
```

Comment:
```suggestion
                b.push_bind(row.expires_at.timestamp());
```
without the `.timestamp()` sqlx will convert this into a datetime string, and jam it into the INTEGER column, which SQLite happily accepts.

Reply:
Why is this just a problem for this column? Why don't we have that issue for `remove_at` and `expire_at`?

Reply:
We do have issues with those columns as well. They currently get isoformat strings stored despite having an integer storage type.